src/main/scala/mastermind/Mastermind.scala

```diff
@@ -13,19 +13,29 @@ val validColors = List('B', 'Y', 'R', 'G')
 
 /** Get a random color from the list of valid colors */
 def getRandomColor(): Color =
-  ???
+  val rand = new scala.util.Random
```

Comment:
It is interesting that you initialized a new Random object by writing the package name/source with it. The one difference I had to this getRandomColor code is that I used the between function instead of nextInt to get an index without using mod.